### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by kernc**

### CODE REVIEW

File: orangecontrib/text/widgets/owcorpusviewer.py [hunk 1]
```diff
@@ -224,3 +229,12 @@ def commit(self):
 
     def _invalidate_selection(self):
         self.commit()
+
+
+if __name__ == '__main__':
+    app = QApplication([])
+    widget = OWCorpusViewer()
+    widget.show()
+    corpus = Corpus.from_file('../datasets/bookexcerpts.tab')
```
Comment (nikicc) [inline orangecontrib/text/widgets/owcorpusviewer.py]: This can be simplified to:
```py
corpus = Corpus.from_file('bookexcerpts')
```

**Comment by nikicc** [comment]
While you're at this, can you disable highlighting for feature names (like `category` and `text` in bookexcerpts)?

**Comment by kernc** [comment]
All text, including feature names, is an HTML which is scanned for matches. I don't see how this would be possible with the current implementation.

**Comment by nikicc** [comment]
I tgought that might be a problem. Anyway, it looks fine to me and also it seems to be quite faster 😄

**Pull request merged**